File: hw/CEDENOR/U3/HW29Composite&TemplateMethod/Python/Composite/controller/Manager.py
import sys
import os
# Nahomi Cedeño, codecreators, DCCO - ESPE

sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
from controller.Supervisor import Supervisor


class Manager(Supervisor):
    def __init__(self, name):
        super().__init__(name)
        self.title = "Manager"

File: hw/CEDENOR/U3/HW29Composite&TemplateMethod/Java/CompositePattern/src/main/java/ec/edu/espe/compositepattern/controller/Manager.java
package ec.edu.espe.compositepattern.controller;

/**
 *
 * @author Nahomi Cedeño, codecreators, DCCO - ESPE
 */
public class Manager extends Supervisor {
    public Manager(String aName) {
        this();
        name = aName;
    }

    public Manager() {
        super();
        title = "Manager";
    }

    @Override
    public void stateName() {
        super.stateName();
    }
}

